packages/flutter/test/widgets/app_test.dart

```diff
@@ -3,9 +3,23 @@
 // found in the LICENSE file.
 
 import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
 import 'package:flutter/widgets.dart';
 import 'package:flutter_test/flutter_test.dart';
 
+class TestAction extends Action {
+  TestAction() : super(key);
+
+  static const LocalKey key = ValueKey<Type>(TestAction);
+
+  int calls = 0;
+
+  @override
+  void invoke(FocusNode node, Intent intent) {
+    calls++;
```

Comment:
Uber style guide nit:
```suggestion
    calls += 1;
```